mongodb_consistent_backup/Upload/S3/S3.py

```diff
@@ -107,9 +107,13 @@ def run(self):
                     part_count += 1
                 if part_count == chunk_count:
                     self._multipart.complete_upload()
-                    key = self.bucket.get_key(key_name)
                     if self.s3_acl:
-                        key.set_acl(self.s3_acl)
+                        try:
+                            self.bucket.set_acl(self.s3_acl, key_name)
+                        except Exception as ex:
+                            logging.warn("Unable to set ACLs on uploaded key. Exception: {}".format(str(ex)))
+                            import traceback
```

Comment:
Is it possible to use logging.exception() here instead of importing+using the traceback module? I believe this will arrive at the same result.

https://docs.python.org/3/library/logging.html#logging.Logger.exception